apps/admin/frontend/config/jest/cssTransform.js

```diff
@@ -5,7 +5,7 @@
 
 module.exports = {
   process() {
-    return 'module.exports = {};';
+    return { code: 'module.exports = {};' };
```

Comment:
Newer jest versions expect an object with `code` property (and presumably optional `sourcemap`).